recipes/cm-unicode/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cm-unicode" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://kent.dl.sourceforge.net/project/{{ name }}/{{ name }}/{{ version }}/{{ name }}-{{ version }}-ttf.tar.xz
+  sha256: 2609c14450f42d0bcd40203900afcb1d693521a9b24a18c65e14b6b0585ff150
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  host:
+    - matplotlib
+
+test:
+  imports:
+    - matplotlib
```

Comment:
This should be a simple existence test that the file is in the right place. See https://conda-forge.org/docs/maintainer/adding_pkgs.html#simple-existence-tests